django_tables2_column_shifter/tables.py

```diff
@@ -35,6 +37,13 @@ def get_column_default_show(self):
         else:
             return self.column_default_show
 
+    def set_hideable_columns(self, hideable_columns):
+        """ Sets columns that are hideable """
+        self.hideable_columns = []
+        for c in self.columns.iterall():
```

Comment:
A good practice is not to use a one-letter variable. 
Code should be self explained.
I suggest to change `c` to `col` or `column`